lib/core/widgets/header_with_address.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:deus/core/util/crypto_util.dart';
+import 'package:deus/core/widgets/svg.dart';
+import 'package:deus/statics/styles.dart';
+import 'package:flutter/material.dart';
+
+class HeaderWithAddress extends StatelessWidget {
+  final String walletAddress;
+  HeaderWithAddress({this.walletAddress});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Container(
+          padding: const EdgeInsets.symmetric(horizontal: 10, vertical: 5),
+          decoration: BoxDecoration(
+              border: Border.all(color: Color(0xFF61C0BF).withOpacity(0.5)),
+              color: Color(0xFF61C0BF).withOpacity(0.25),
```

Comment:
Auch wenn es nur hier vorkommt, macht es vermutlich Sinn, das in my_colors.dart als konstante festzuhalten.

- [ ] `kAddressBackground`

Reply:
und dann einmal eben im Repo schauen, ob du sonst noch irgendwo 0xFF61C0BF verwendest und ggf. ersetzen